libecl/src/ecl_grid.c

```diff
@@ -1197,33 +1197,29 @@ struct tetrahedron_struct {
     point_type p3;
 };
 
+static inline void point_isub(point_type* a, const point_type* b) {
+  a->x -= b->x;
+  a->y -= b->y;
+  a->z -= b->z;
+}
+
+static inline double determinant(const point_type* a,
```

Comment:
If possible I would prefer if you could split the determinant based volume calculation in a separate PR; then we can check if it triggers an update of the opm reference data.